src/cpu.rs

```diff
@@ -52,97 +51,97 @@ pub(crate) fn features() -> Features {
                     let [l1edx, l1ecx, l7ebx, l7ecx] = unsafe { &mut GFp_ia32cap_P };
```

Comment:
I saw later in this detection that it requires "feature = sgx" -- maybe just "target_env = sgx" is enough?

Reply:
Yeah most likely is enough, since we're using the `sgx` feature by default. I will change it.

Reply:
yeah, perhaps no need for sgx feature, if it can be guarded with `target_env`